test/gpu/native/amd/extern_kernel_launch.chpl

```diff
@@ -0,0 +1,91 @@
+extern {
+  #include <hip/hip_runtime.h>
+  #include <hip/hip_runtime_api.h>
+  #include <assert.h>
+
+  extern char* chpl_gpuBinary;
+
+  #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+
+  static double launchKernel(){
+    hipDevice_t    device;
+    hipModule_t    module;
+    hipFunction_t  function;
+    int            devCount;
+    hipCtx_t       context = NULL;
+
+    double X;
+
+    hipCtxGetCurrent(&context);
+
+    if (context == NULL) {
+      hipInit(0);
+      hipGetDeviceCount(&devCount);
+      hipDeviceGet(&device, 0);
+
+      char name[128];
+      hipDeviceGetName(name, 128, device);
+      hipCtxCreate(&context, 0, device);
+    }
+
+    /*
```

Comment:
Does this mean the fatbin data is getting embedded into `chpl_gpuBinary` (like we do with nvidia). If so cool. You can remove this commented out section of code.